hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/FinalizeChildRecordHandler.java

```diff
@@ -66,6 +67,8 @@ public void finalizeChildRecord(@NonNull final ChildFinalizeContext context) {
             recordBuilder.transferList(TransferList.newBuilder()
                     .accountAmounts(asAccountAmounts(hbarChanges))
                     .build());
+        } else if (recordBuilder.status() != ResponseCodeEnum.SUCCESS) {
+            recordBuilder.transferList(null);
```

Comment:
`finalizeChildRecord` feels like a weird place to make sure the _user transaction_ transfer list has nothing in it. I'm not sure a better place for it though 🤔

Reply:
In `mono` when there are multiple failing child records we observe nulled out fields like `transferList` while in `mod` we provide the default values. This difference (mono null vs mod empty objet)  caused the fuzzy-match to fail so I added this change to match the mono behaviour.

Reply:
Nevermind, I was thinking that there was also a `childRecordBuilder` available